### PR TITLE
feat: add interfaces report to support bundle

### PR DESCRIPTION
**Pull request by spikecurtis**
Adds the `InterfacesReport` to the `coder support bundle`

### CODE REVIEW

**Comment by spikecurtis** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/coder/coder/13563?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#13563** <a href="https://app.graphite.dev/github/pr/coder/coder/13563?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#13562** <a href="https://app.graphite.dev/github/pr/coder/coder/13562?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @spikecurtis and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: codersdk/healthsdk/interfaces.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+			if addr.Addr().IsLinkLocalUnicast() || addr.Addr().IsLinkLocalMulticast() {
+				continue
+			}
+			r = true
+		}
+		if !r {
+			continue
+		}
+		report.Interfaces = append(report.Interfaces, healthIface)
+		if iface.MTU < 1378 {
```
Comment (johnstcn) [inline codersdk/healthsdk/interfaces.go:64]: Probably a good idea to extract this to a variable with a comment as to why this particular value is important.

**Review by johnstcn — APPROVED** [review]
LGTM but hasn't this already been merged? 
https://github.com/coder/coder/pull/13562/files

**Comment by spikecurtis** [comment]
https://github.com/coder/coder/pull/13562/files is downstack.  Because we squash & merge, PRs up the stack can look like they include downstack changes.  Lemme restack and push and update...

**Comment by spikecurtis** [comment]
### Merge activity

* **Jun 13, 5:09 AM EDT**: @spikecurtis merged this pull request with [Graphite](https://app.graphite.dev/github/pr/coder/coder/13563).

**Pull request merged**